schematic/schemas/explorer.py

```diff
@@ -184,7 +184,17 @@ def get_adjacent_nodes_by_relationship(
         return list(nodes)
 
     def is_class_in_schema(self, class_label):
-        if self.schema_nx.nodes[class_label]:
+        
+        # accessing the node in the dictionary will error out if the class is not in the schema
```

Comment:
so sorry I forgot you can do `if class_label in self.schema_nx.nodes()` and that will give True if the class label is in the schema and false if not, the you can raise the helpful error.